src/utils/formDataUtils.ts

```diff
@@ -25,7 +25,7 @@ const errorMessages = {
     "uppercase and lowercase letters, include a mixture of letters and " +
     "numbers, and have at least one special character except period (.)",
```

Comment:
Maybe when we go over the code base next week we can spend some time here.. I am wondering if we add now error messages in multiple places?

Reply:
Yeah, it's confusing. Especially because this PR actually doesn't cover some of the no-js server-side error messages.